### PR TITLE
Fix/user avatar themes master

### PR DESCRIPTION
**Pull request by nistormihai**

### CODE REVIEW

File: server/liveblog/themes/themes_assets/angular/theme.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "label": "Angular Liveblog Toolkit",
     "name": "angular",
     "abstract": true,
-    "version": "1.4.38",
+    "version": "3.0.1",
```
Comment (m0g) [inline server/liveblog/themes/themes_assets/angular/theme.json]: 3.0.1 ?

**Comment by m0g** [comment]
Guys, what version is this fix targeting?

File: server/liveblog/themes/themes_assets/angular/theme.json (hunk 1 above)
Comment (nistormihai) [inline server/liveblog/themes/themes_assets/angular/theme.json]: yes after talks with Gideon, it was decided that is the best to increase the major version to match the liveblog repo.
Also an increase on minor version was omitted, in the future we need to release also the `angular`, `classic` and now `default` themes on liveblog release.

**Comment by nistormihai** [comment]
You mean what liveblog version? it should be on the next release version.

**Pull request merged**